pkg/infoschema/builder.go

```diff
@@ -177,7 +178,8 @@ func (b *bundleInfoBuilder) updateTableBundles(is *infoSchema, tableID int64) {
 
 // Builder builds a new InfoSchema.
 type Builder struct {
-	is *infoSchema
+	enableV2 bool
```

Comment:
IMHO, we can have a brand new builder for V2. Since many schema update work messed up with v1 and v2 now. Some of the structures should not updated for v2 may be updated by accident during developing.

Ok for now.

Reply:
Yes, I agree with the brand new builder for V2, but it involves much more code changes.
At the current phase we can keep the minimal code changes and go ahead.